templates/openstack/install.html

```diff
@@ -39,105 +38,137 @@ <h1>Install OpenStack yourself</h1>
 </section>
 
 <section class="p-strip--light is-bordered">
-  <div class="row">
-    <div class="col-8">
-      <h2>Single-node, multi-node or large scale?</h2>
-      <p><strong>Choose the OpenStack installation option that suits you best:</strong></p>
+  <div class="u-fixed-width">
+    <h2>Single-node, multi-node or large-scale cluster?</h2>
+    <p><strong>Choose the OpenStack installation option that suits you best:</strong></p>
+  </div>
+  <div class="p-tabs">
+    <div class="u-fixed-width">
+      <div class="p-tabs__list" role="tablist" aria-label="openstack-deployment">
+        <div class="p-tabs__item">
+          <button style="background-color: transparent;" class="p-tabs__link" role="tab" aria-selected="true" aria-controls="single-node-deployment" id="single-node-deployment-tab">Single&dash;node deployment</button>
+        </div>
+        <div class="p-tabs__item">
+          <button style="background-color: transparent;" class="p-tabs__link" role="tab" aria-selected="false" aria-controls="multi-node-deployment" id="multi-node-deployment-tab" tabindex="-1">Multi&dash;node deployment</button>
+        </div>
+        <div class="p-tabs__item">
+          <button style="background-color: transparent;" class="p-tabs__link" role="tab" aria-selected="false" aria-controls="large-scale-deployment" id="large-scale-deployment-tab" tabindex="-1">Large-scale deployment</button>
+        </div>
+      </div>
     </div>
   </div>
-  <div class="row u-equal-height">
-    <div class="col-4 p-card">
-      <h3 class="p-card__title">Single&dash;node deployment</h3>
-      <p class="p-card__content">Suitable for testing and development use cases:</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">One physical machine needed</li>
-        <li class="p-list__item is-ticked">Uses MicroStack &ndash; OpenStack in a snap</li>
-        <li class="p-list__item is-ticked">Core services included</li>
-        <li class="p-list__item is-ticked">All services on a single node</li>
-        <li class="p-list__item is-ticked">Opinionated OpenStack</li>
-        <li class="p-list__item is-ticked">Straightforward installation</li>
-        <li class="p-list__item is-ticked">OpenStack "on-rails"</li>
-      </ul>
-      <p><a href="#single-node-deployment">Single-node installation instructions &rsaquo; Deploy OpenStack on a single node&nbsp;&rsaquo;</a></p>
+  <div class="u-fixed-width">
+    <div tabindex="0" role="tabpanel" id="single-node-deployment" aria-labelledby="single-node-deployment">
+      <div class="row">
```

Comment:
If you change the div on line 60 to a row, you dont need the row div on line 62.